### PR TITLE
Added default volumes

### PR DESCRIPTION
**Pull request by Vj-L**
I worked on the #6770  providing the default volumes for each of the containers 

#
### Type of change :

- [x] Bug fix

### CODE REVIEW

**Review by harshach — APPROVED** [review]

File: docker/run_local_docker.sh [hunk 1]
```diff
@@ -55,20 +55,21 @@ else
...
 
-cd docker/local-metadata || exit
+#cd docker/local-metadata || exit
 
 echo "Stopping any previous Local Docker Containers"
-docker compose -f docker-compose-postgres.yml down
+docker compose  -f docker/local-metadata/docker-compose-postgres.yml down
 docker compose down
-
+rm -rf $PWD/docker-volume
```
Comment (akash-jain-10) [inline docker/run_local_docker.sh:63]: Add a condition to run only if the `docker-volume` directory exists!

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]
All the `docker-compose*` looks good to me. A minor condition to add in `run_local_docker.sh` script.

File: docker/run_local_docker.sh (hunk 1 above)
Comment (Vj-L) [reply docker/run_local_docker.sh]: done

**Comment by harshach** [comment]
@Vj-L what is remaining here to merge in. It looks like your diff brought some other files that are not relevant to this PR?

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**